Application/EdFi.Ods.Tests/EdFi.Ods.Security/Authorization/Repositories/EntityAuthorizerTests.cs

```diff
@@ -1,90 +0,0 @@
-﻿// SPDX-License-Identifier: Apache-2.0
-// Licensed to the Ed-Fi Alliance under one or more agreements.
-// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
-// See the LICENSE and NOTICES files in the project root for more information.
-
-using EdFi.Common.Extensions;
-using EdFi.Ods.Api.Security.Authorization;
-using EdFi.Ods.Api.Security.Authorization.Filtering;
-using EdFi.Ods.Api.Security.Authorization.Repositories;
-using EdFi.Ods.Common.Exceptions;
-using EdFi.Ods.Common.Infrastructure.Filtering;
-using EdFi.Ods.Common.Models.Domain;
-using EdFi.Ods.Common.Security.Authorization;
-using EdFi.Ods.Common.Security.Claims;
-using EdFi.TestFixture;
-using FakeItEasy;
-using NUnit.Framework;
-using Shouldly;
-using System.Threading;
-using Test.Common;
-
-namespace EdFi.Ods.Tests.EdFi.Ods.Api.Security.Authorization.Repositories
-{
-    [TestFixture]
-    public class EntityAuthorizerTests
-    {
-        public class When_authorizing_an_unauthorized_caller: ScenarioFor<EntityAuthorizer>
-        {
-            private const long ExpectedClaimEndpointValue = long.MaxValue;
```

Comment:
Removed this unit test since it's now being tested in Postman [here](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/blob/c4d330173271dc7d0a36874434653255c7064f1c/Postman%20Test%20Suite/Ed-Fi%20ODS-API%20Integration%20Test%20Suite.postman_collection.json#L12811).